lib/logster/cache.rb

```diff
@@ -0,0 +1,16 @@
+module Logster
+  class Cache
```

Comment:
when we implement one for core we also need a #clear hear. and to make sure #clear is being called when amending stuff via the UX.